### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Goruchie**
# **Add the following features:**
- [x] **Create a method called current_user in your ApplicationController to make current user data available to all controllers.**
- [x] **Create forms to perform the following functions:**
- Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
- Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- [x] Allow Users to add likes to Posts.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @Goruchie 🙋‍♀️,  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project, that has a README that does not meet all the requirements.

- Please delete the live demo section since the application is not deployed. If project requirements say you should deploy, then kindly restore the Live Demo section and put the link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Goruchie** [comment]
@KDlamini Thanks for letting me know that, I've already fixed that, and now my project should be reviewed!

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    if @comment.save
+      redirect_to user_posts_path(current_user)
+    else
+      render :new
+    end
+  end
```
Comment (omarsalem7) [inline app/controllers/comments_controller.rb:13]: - [ ] When I try to make a comment, I get the following error, kindly fix this issue and make sure adding comments works properly.
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Goruchie 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Goruchie** [comment]
Hello @omarsalem7! Thanks for the review! I've checked the change required and I just changed it, so now my project should be approved!

**Review by mwanawabangona — APPROVED** [review]
Hi @Goruchie,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested change!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**